build/fpga-regmap/src/lib.rs

```diff
@@ -33,11 +38,31 @@ enum Node {
 }
 
 #[derive(Debug, Deserialize)]
-struct EnumEncode {
+pub struct EnumEncode {
     name: String,
     value: u8,
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum SwAccess {
```

Comment:
As a note, there are a few more possible types here: https://systemrdl-compiler.readthedocs.io/en/stable/api/types.html#systemrdl.rdltypes.builtin_enums.AccessType
that could show up.

We also don't do much with the OnRead and OnWrite types described below that section but am open to doing so if there's code-gen value.

Reply:
Interesting, I'm going to leave those as unimplemented so that they'll fail loudly if we ever end up using them!